changes/probe.py

```diff
@@ -58,11 +68,20 @@ def has_changelog():
 
 
 def has_readme():
-    """README.md"""
+    """README"""
+    readme = None
+    message = 'Create a README file'
+    for filename in os.listdir("."):
+        readme = re.match(r"README($|\.\w{2,})", filename, re.I)
+        if readme:
+            if readme.group(1) not in README_EXTENSIONS:
+                message += ' with a valid extension (or none)'
+                readme = None
+            break
```

Comment:
This could be clearer if you broke it up into `locate_readme` and `validate_readme` stanzas/helper functions?